code/modules/surgery/organs/augments_arms.dm

```diff
@@ -231,7 +262,26 @@
 		var/obj/item/assembly/flash/armimplant/F = locate(/obj/item/assembly/flash/armimplant) in items_list
 		F.I = src
 
-/obj/item/organ/cyberimp/arm/surgery
-	name = "surgical toolset implant"
-	desc = "A set of surgical tools hidden behind a concealed panel on the user's arm."
-	contents = newlist(/obj/item/retractor/augment, /obj/item/hemostat/augment, /obj/item/cautery/augment, /obj/item/surgicaldrill/augment, /obj/item/scalpel/augment, /obj/item/circular_saw/augment, /obj/item/surgical_drapes)
+/obj/item/organ/cyberimp/arm/esword
+	name = "arm-mounted energy blade"
+	desc = "An illegal and highly dangerous cybernetic implant that can project a deadly blade of concentrated energy."
+	contents = newlist(/obj/item/melee/transforming/energy/blade/hardlight)
+
+/obj/item/organ/cyberimp/arm/shield
+	name = "arm-mounted riot shield"
+	desc = "A deployable riot shield to help deal with civil unrest."
+	contents = newlist(/obj/item/shield/riot/implant)
+
+/obj/item/organ/cyberimp/arm/shield/emag_act()
+	. = ..()
+	if(locate(/obj/item/assembly/flash/armimplant) in items_list)
+		return
+	to_chat(usr, "<span class='notice'>You unlock [src]'s integrated knife!</span>")
+	items_list += new /obj/item/assembly/flash/armimplant(src)
+	return TRUE
+
+/obj/item/organ/cyberimp/arm/shield/Initialize()
+	. = ..()
+	if(locate(/obj/item/assembly/flash/armimplant) in items_list)
+		var/obj/item/assembly/flash/armimplant/F = locate(/obj/item/assembly/flash/armimplant) in items_list
+		F.I = src
```

Comment:
That is the baddest copypasta of this entire PR. Replace with the following.
```
	var/obj/item/assembly/flash/armimplant/F = new(src)
	F.I = src
	items_list += F
	return TRUE
```

Reply:
I did it correctly right?